exceptions/guild.py

```diff
@@ -0,0 +1,43 @@
+"""Custom exceptions for guild related cogs."""
```

Comment:
Incorrect description of exceptions held within `exceptions/guild.py`. Improved description:

```python
"""Custom exception classes to be raised when errors occur with access properties of a Discord guild object."""
```